src/Service/CloudFront/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "async-aws/cloud-front",
+    "type": "library",
+    "description": "CloudFront client, part of the AWS SDK provided by AsyncAws.",
+    "keywords": [
+        "aws",
+        "amazon",
+        "sdk",
+        "async-aws",
+        "cloud-front"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": "^7.2.5",
+        "ext-json": "*",
```

Comment:
it looks like it don't use json, but xml

Reply:
Thank yuo